lib/cacheable/response_cache_handler.rb

```diff
@@ -140,7 +140,7 @@ def serve_from_cache(cache_key_hash, cache_age_tolerance=nil, message = "Cache h
               body = Cacheable.decompress(body)
             end
 
-            render text: body, status: status
+            render plain: body, status: status
```

Comment:
The gemspec, so in that case actionpack 5.1.2. Maybe we should test with `actionpack >4` and `actionpack 5` ? https://github.com/Shopify/shopify-cloud/tree/master/gemfiles